h/static/scripts/tests/util/submit-form-test.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+var submitForm = require('../../util/submit-form');
+
+/**
+ * Fake `fetch()` implementation.
+ *
+ * This does not mock the global `window.fetch` function but provides a mock
+ * via `this.fetch` which can be passed as a dependency to functions that make
+ * network requests.
+ */
+class FakeFetch {
```

Comment:
A caveat is that sinon's stubbing toolkit doesn't work if we ever run our tests in a browser which _does_ support fetch natively, such as headless Chrome for example.

I personally think that faking the Fetch API directly, but re-using the real [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response/Response) class, is nicer, as it involves less indirection/magic.